.github/scripts/generate_ci_workflows.py

```diff
@@ -138,10 +138,12 @@ class CIWorkflow:
     only_build_on_pull_request: bool = False
     only_run_smoke_tests_on_pull_request: bool = False
     num_test_shards_on_pull_request: int = -1
+    distributed_test: bool = True
```

Comment:
currently set as default true just to make sure we don't exclude any existing tests we are currently doing